test/Ocelot.UnitTests/Configuration/Validation/FileConfigurationFluentValidatorTests.cs

```diff
@@ -11,7 +11,10 @@
 using Ocelot.UnitTests.Requester;
 using Ocelot.Values;
 using System.Security.Claims;
-using System.Text.Encodings.Web;
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration;
+using Ocelot.Logging;

```

Comment:
Note 2 Order

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/1753/commits/6504e3ef6d3c3a1d70a57bc9100f8f7397fa427a